bot/util/utilities.py

```diff
@@ -154,24 +154,145 @@ async def cembed(self, ctx, channel: str, *, json_string: str):
         except discord.errors.HTTPException:
             await ctx.send("**__Error:__** Could not parse json. Make sure your last argument is valid JSON.")
         except discord.DiscordException:
-            await ctx.send(
-                "**__Error:__** Invalid embed. Make sure you have at least title or description set (also for each additional field). You can validate your json at https://leovoel.github.io/embed-visualizer/.")
+            await ctx.send("**__Error:__** Invalid embed. Make sure you have at least title or description set (also "
+                           "for each additional field). You can validate your json at "
+                           "https://leovoel.github.io/embed-visualizer/.")
         except ValueError as error:
             await ctx.send(error)
         except TypeError:
             await ctx.send("**__Error:__** Error creating embed. Please check your parameters.")
 
     @commands.command(name="echo")
-    async def echo(self, ctx, channel: typing.Optional[discord.TextChannel], *, text: str):
+    @commands.has_guild_permissions(administrator=True)
```

Comment:
is moderator an administrator?

Reply:
No. But right now I'm thinking about if it would be a good idea making this command available to Mods too.